Sources/Defaults/Defaults.swift

```diff
@@ -36,10 +35,12 @@ public enum Defaults {
 
 	public final class Key<Value: Serializable>: AnyKey {
 		public let defaultValue: Value
+		/// A flag to determine whether CodableBridge is a first-class serializator
+		var usingCodable = true
 
 		/// Create a defaults key.
 		/// The `default` parameter can be left out if the `Value` type is an optional.
-		public init(_ key: String, default defaultValue: Value, suite: UserDefaults = .standard) {
+		public init(_ key: String, default defaultValue: Value, suite: UserDefaults = .standard, usingCodable: Bool = true) {
```

Comment:
I don't the parameter should be available when the value is not `Codable`. It wouldn't work anyway.